CHANGELOG.md

```diff
@@ -12,4 +12,14 @@ the following form: YYYY.0M.0D.
 
 - Designed a InventorySystem component
 - Designed a HealthSystem component
-- Designed a LevelingSystem component
\ No newline at end of file
+- Designed a LevelingSystem component
+
+## 2024.10.03
```

Comment:
It seems weird, but we usually list the dates in reverse chronological order. That way, the latest change is first.